packages/angular/pwa/BUILD.bazel

```diff
@@ -4,28 +4,33 @@
 # found in the LICENSE file at https://angular.dev/license
 
 load("@npm//@bazel/jasmine:index.bzl", "jasmine_node_test")
-load("//tools:defaults.bzl", "pkg_npm")
-load("//tools:interop.bzl", "ts_project")
+load("//tools:defaults2.bzl", "npm_package", "ts_project")
 load("//tools:ts_json_schema.bzl", "ts_json_schema")
 
 licenses(["notice"])
 
 package(default_visibility = ["//visibility:public"])
 
+RUNTIME_ASSETS = glob(
+    include = [
+        "pwa/*.js",
+        "pwa/*.mjs",
+        "pwa/files/**/*.png",
```

Comment:
NIT: I’d glob all the file under files as if a new file is added it’s easy to forget to add it.